root/etc/redis-rspamd.conf

```diff
@@ -0,0 +1,57 @@
+bind 127.0.0.1
```

Comment:
is it still required if `unixsocket` is also specified?

Reply:
when commented, do systemctl status
```  Main PID: 11731 (redis-server)
CGroup: /system.slice/redis-rspamd.service
           └─11731 /usr/bin/redis-server *:0```
or if not commented

 ```Main PID: 11778 (redis-server)
   CGroup: /system.slice/redis-rspamd.service
           └─11778 /usr/bin/redis-server 127.0.0.1:0```

I do prefer to bind to localhost, eventy if `port 0` disable TCP socket